experiment/simulation/css/main.css

```diff
@@ -1 +1,147 @@
-/* You CSS goes in here */
\ No newline at end of file
+.instructions-box {
+  margin: auto;
+  margin-top: 1%;
+  max-width: 80%;
+  padding: 1% 1%;
+}
+
+.procedure-message {
+  font-size: 120%;
+}
+
+.workspace {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
```

Comment:
Why do we need this class? The display and flex-direction properties are covered by other classes on the same element.